docs/src/gettingstarted/developing/dev-environments/create-environment.md

```diff
@@ -34,7 +34,7 @@ Platform.sh supports the deployment of isolated development environments from yo
 
```

Comment:
The [note is not displayed properly](https://pr-2203-kwyjlpq-652soceglkw4u.eu-3.platformsh.site/gettingstarted/developing/dev-environments/create-environment.html).

This 🐛  is already present in the [current version of the docs](https://docs.platform.sh/gettingstarted/developing/dev-environments/create-environment.html).

It seems to be caused by the indentation taking precedence over the `{{< note >}}`.
Removing the indentation for the note fixes the issue (but makes it poorly integrated into the flow).

Reply:
The problem was using 4 spaces instead of 3. 4 indicates a code block (it's an alternative to the ``` style) and 3 indicates alignment with the number for the ordered list.